pom.xml

```diff
@@ -149,7 +149,7 @@
         <version.auto-value>1.10</version.auto-value>
         <version.error-prone>${version.error-prone-orig}</version.error-prone>
         <version.error-prone-fork>v${version.error-prone-orig}-picnic-3</version.error-prone-fork>
-        <version.error-prone-orig>2.15.0</version.error-prone-orig>
+        <version.error-prone-orig>2.16</version.error-prone-orig>
```

Comment:
This also requires a new release for our fork; I started that process.

Reply:
I managed to [trigger a successful build](https://jitpack.io/com/github/PicnicSupermarket/error-prone/error_prone_annotation/v2.16.0-picnic-1/error_prone_annotation-v2.16.0-picnic-1.pom), but the artifacts are not (yet?) available. Let's see whether this is a systems issue that will resolve itself.